web/docs/operators/sources/api.md

```diff
@@ -0,0 +1,46 @@
+# api
+
+Use Tenzir's REST API directly from within a pipeline.
+
+## Synopsis
+
+```
+api <endpoint> [<key=value>...]
+```
+
+## Description
+
+The `api` operator interacts with Tenzir's REST API without needing to spin up a
+web server, making all APIs accessible from within pipelines.
+
+:::info OpenAPI
+Visit [Tenzir's REST API specification](/api) to see a list of all available
+endpoints.
+:::
+
+### `<endpoint>`
+
+The endpoint to query, e.g., `/pipeline/list` to list all pipelines created
```

Comment:
```suggestion
The node to query, e.g., `/pipeline/list` to list all pipelines created
```

Reply:
This doesn't make sense, it's an endpoint we're sending a request to. I changed the verb to _request_ to clarify this.